packages/cadl-ranch-specs/http/models/dictionary/main.cadl

```diff
@@ -0,0 +1,53 @@
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+using Azure.Core.Foundations;
+
+@serviceTitle("Dictionary in Complex Test Service")
+@serviceVersion("2016-02-29")
+@server("http://localhost:3000", "Testserver endpoint")
+@doc("Illustrates dictionary generation, as well as serialization and deserialization.")
+namespace Cadl.Testserver.BodyComplexDictionary;
+
+model dictionaryWrapper {
+  defaultProgram?: Record<string>;
```

Comment:
`defaultProgram` isn't a good property name, we can do `optionalDictionaryProperty` (and maybe even add a `requiredDictionaryProperty`.